js_task1/thirdTask.js

```diff
@@ -0,0 +1,151 @@
+const input = document.querySelector("input");
+const getMoneyBtn = document.querySelector("button");
+const infoBlock = document.querySelector(".info");
+
+class ATM {
+  userValue = 0;
+  maxStoreValue = 0;
+  moneyToOut = 0;
+  outNotes = [];
```

Comment:
It is not clear what mean outNotes. Please rename it

Reply:
done